### PR TITLE
Feature browse

### PR DESCRIPTION
**Pull request by rider3458**
Use GET method to apply changes based on the chosen options
Default browse option is "all categories".

### CODE REVIEW

File: private/browse.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    *
+    */
+
+    /**
+    * Retrieve store name and store, then insert them into the store card
+    * @param $store_name
+    * @param $store_id
+    */
+    function display_store($store_name, $store_id) {
+        $store_href = url_for("/store/store-template/?={$store_id}");
```
Comment (doanyennhi) [inline private/browse.php:16]: It should be "/store/store-template/?id={$store_id}"

File: private/browse.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        echo "
+            <div class='store-card'>
+                <a href='$store_href'><img class='store-card-thumbnail' alt='image representation of a shop' src='../../media/image/placeholder_262x250.png'></a>
```
Comment (doanyennhi) [inline private/browse.php:19]: Can you use the profile placeholder 143x143 here cause Thu and I are using that

**Review by doanyennhi — CHANGES_REQUESTED** [review]

File: private/browse.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+    /*
+    *
+    * Yabe Comma-separated values database interface library.
```
Comment (miketvo) [inline private/browse.php:5]: You didn't change this line :) This is not a CSV interface library. It's a database browsing interface library

**Review by miketvo — CHANGES_REQUESTED** [review]
@rider3458 
Good coding style! But you didn't implement browsing for products. Please implement it.

Also, check out the comment below.

🐶

**Review by miketvo — APPROVED** [review]

**Review by doanyennhi — APPROVED** [review]

**Pull request merged**